### PR TITLE
Wired up UI to Export Private Keys from a Default HD Wallet

### PR DESCRIPTION
**Pull request by Douglashdaniel**
## Description 

1) Wired up the `Account Settings` modal UI to give the ability to export a `Private Key` from a Default HD Wallet.
(Note: I did not show my private key in video for obvious reasons! 😉 )

Resolves <https://github.com/brave/brave-browser/issues/17769>

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/handbook/blob/master/development/security.md#when-is-a-security-review-needed), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/handbook/blob/master/development/security.md#when-is-a-security-review-needed), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

https://user-images.githubusercontent.com/40611140/131547565-e5a1357e-3c9e-4163-91ee-546fcc295e14.mov

### CODE REVIEW

File: components/brave_wallet_ui/page/async/wallet_page_async_handler.ts [hunk 1]
```diff
@@ -97,8 +97,13 @@ handler.on(WalletPageActions.removeImportedAccount.getType(), async (store, payl
 
 handler.on(WalletPageActions.viewPrivateKey.getType(), async (store, payload: ViewPrivateKeyPayloadType) => {
   const keyringController = (await getAPIProxy()).keyringController
-  const result = await keyringController.getPrivateKeyForImportedAccount(payload.address)
-  store.dispatch(WalletPageActions.privateKeyAvailable({ privateKey: result.privateKey }))
+  if (payload.isDefault) {
+    const result = await keyringController.getPrivateKeyForDefaultKeyringAccount(payload.address)
```
Comment (yrliou) [inline components/brave_wallet_ui/page/async/wallet_page_async_handler.ts:101]: nit: how about 
```
const result = payload.isDefault ? await keyringController.get... : await keyringController.get...
store.dispatch(WalletPageActions.privateKeyAvailable({ privateKey: result.privateKey }))
```

Comment (Douglashdaniel) [reply components/brave_wallet_ui/page/async/wallet_page_async_handler.ts]: Updated!

**Review by yrliou — APPROVED** [review]
LGTM

**Comment by Douglashdaniel** [comment]
> LGTM

Thank you! 🙏

**Pull request merged**